linguistics/src/main/java/com/yahoo/language/opennlp/LangDetectModel.java

```diff
@@ -0,0 +1,13 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.language.opennlp;
+
+import opennlp.tools.langdetect.LanguageDetectorModel;
+
+/**
+ * @author jonmv
+ */
+public interface LangDetectModel {
+
+    LanguageDetectorModel load();
```

Comment:
... and here (especially, what does retuning null mean and can this throw an exception.